scripts/mutmut-notify-on-resume.sh

```
@@ -0,0 +1,36 @@
+IMAGE="${MUTMUT_IMAGE:-python:3.14-bookworm}"
+
+echo "Running mutmut in Docker ($IMAGE) for notify-on-resume code..."
+docker run --rm \
+  -v "$ROOT:/app" \
+  -w /app \
+  "$IMAGE" \
+  bash -lc "
+    set -euo pipefail
+    pip install -q mutmut==3.4.0 pytest pytest-asyncio pytest-homeassistant-custom-component pytest-cov voluptuous
+    pip install -q 'homeassistant>=2024.1'
+    mutmut run --max-children ${MAX_CHILDREN} $(printf '%q ' \"${PATTERNS[@]}\")
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)